pytest_fixtures/component/taxonomy.py

```diff
@@ -160,6 +160,22 @@ def function_entitlement_manifest():
         yield manifest
 
 
+@pytest.fixture(scope='function')
+def duplicate_entitlement_manifest():
+    """Yields a second entitlement manifest that can be used with a function-scoped organization
+    when a test requires two separate manifests."""
+    with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
+        yield manifest
+
+
+@pytest.fixture(scope='function')
+def duplicate_sca_manifest():
```

Comment:
I'm pretty sure if you use just `@pytest.fixture` it will be function scoped. Non-blocking.

Reply:
You're right, function-scoped is the default. I can remove the extraneous scope assignments.